src/main/java/co/omise/Example.java

```diff
@@ -419,8 +419,20 @@ void createSource() throws IOException, OmiseException, ClientException {
 
         Source source = client().sendRequest(request);
         System.out.printf("source created: %s", source.getId());
-}
-  
+    }
+
+    void createSource_installment() throws IOException, ClientException, OmiseException {
```

Comment:
Not sure about mixing `_` in the method name. Could you change to camel-case?

Reply:
Sure